modules/atomic-widgets/styles/styles-renderer.php

```diff
@@ -111,22 +104,11 @@ private function variant_to_css_string( string $base_selector, array $variant ):
 	}
 
 	private function props_to_css_string( array $props ): string {
-		$css = [];
-
-		foreach ( $props as $prop => $raw_value ) {
-			$prop = $this->camel_case_to_dash( $prop );
-			$value = $this->transform_value( $raw_value );
-
-			if ( $prop && $value ) {
-				$css[] = $prop . ':' . $value;
-			}
-		}
+		$schema = Style_Schema::get();
 
-		return implode( ';', $css );
-	}
-
-	private function camel_case_to_dash( string $str ): string {
-		return strtolower( preg_replace( '/([a-zA-Z])(?=[A-Z])/', '$1-', $str ) );
+		return Collection::make( Props_Resolver::for_styles()->resolve( $schema, $props ) )->filter()->map( function ( $value, $prop ) {
+			return $prop . ':' . $value . ';';
+		} )->implode( '' );
```

Comment:
Style change if you want:
```suggestion
		return Collection::make( Props_Resolver::for_styles()->resolve( $schema, $props ) )
			->filter()
			->map( fn ( $value, $prop ) => $prop . ':' . $value . ';' )
			->implode( '' );
```